client/src/components/Preloader/Preloader.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classes from './preloader.module.css';
+import sun from '../../../public/images/sun.png';
+import moon from '../../../public/images/moon.png';
+import PropTypes from 'prop-types';
+
+function Preloader({ staticImage, rotatingImage, theme }) {
+  return (
+    <div className={classes.container} data-theme={theme}>
+      <img src={staticImage} className={classes.staticImage} alt="cental obj" />
```

Comment:
1) Still not correct to my mind. src={imageSrc}
2) So you want to make it flexible and pass some image sources from above... but then again you are using rotating, static prefixes which indicate that they are bound to specific look. Not good. Either hardcode them here or make it really flexible